probe/docker/container.go

```diff
@@ -389,20 +389,18 @@ func (c *container) getBaseNode() report.Node {
 	return result
 }
 
-func (c *container) controlsMap() map[string]report.NodeControlData {
+// Return a slice including all controls that should be shown on this container
+func (c *container) controls() []string {
 	paused := c.container.State.Paused
-	running := !paused && c.container.State.Running
-	stopped := !paused && !running
-	return map[string]report.NodeControlData{
-		UnpauseContainer: {Dead: !paused},
-		RestartContainer: {Dead: !running},
-		StopContainer:    {Dead: !running},
-		PauseContainer:   {Dead: !running},
-		AttachContainer:  {Dead: !running},
-		ExecContainer:    {Dead: !running},
-		StartContainer:   {Dead: !stopped},
-		RemoveContainer:  {Dead: !stopped},
+	switch {
+	case paused:
```

Comment:
Nit: I guess I do have a little bit of OCD:

```suggestion
	case c.container.State.Paused:
```